src/kirby/components/slide-button/slide-button.component.tns.scss

```diff
@@ -3,18 +3,21 @@
 .slide-button-container {
```

Comment:
Duplicated styling - consider creating a shared scss

Reply:
Only the variables and selectors was duplicated. Moved what was duplicated to common scss

Reply:
Done